autogluon_zeroshot/utils/rank_utils.py

```diff
@@ -40,7 +40,7 @@ def get_rank(error: float,
         else:
             win = True
         if win:
-            if include_partial and error != 0:
+            if include_partial and error > 0:
```

Comment:
This is a tangential change, previously we could get very strange ranks if error was somehow negative. This ensures we can't get ranks below 0.